pkg/operator/ceph/cluster/cephstatus.go

```diff
@@ -163,6 +165,20 @@ func toCustomResourceStatus(currentStatus cephv1.ClusterStatus, newStatus *cephc
 			Message:  message.Summary.Message,
 		}
 	}
+
+	if newStatus.PgMap.TotalBytes != 0 {
+		s.Capacity.TotalBytes = newStatus.PgMap.TotalBytes
+		s.Capacity.UsedBytes = newStatus.PgMap.UsedBytes
+		s.Capacity.AvailableBytes = newStatus.PgMap.AvailableBytes
+		s.Capacity.LastUpdated = formatTime(time.Now().UTC())
+		previousCheck = true
+	}
+
+	if previousCheck && newStatus.PgMap.TotalBytes == 0 {
+		s.Capacity = currentStatus.CephStatus.Capacity
+		s.Capacity.LastUpdated = formatTime(time.Now().UTC())
```

Comment:
We want to preserve the same timestamp as previously, so we don't need this line. We only want to update the timestamp when we successfully queried for the usage.

Reply:
Done.